kubernetes/blockscout/deployment.yml

```diff
@@ -0,0 +1,85 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: blockscout
+  labels:
+    app: astria-dev-cluster
+  namespace: astria-dev-cluster
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: astria-dev-cluster
+  template:
+    metadata:
+      name: blockscout-local
+      labels:
+        app: astria-dev-cluster
+    spec:
+      containers:
+        - name: blockscout
+          command: ["bash"]
+          args:
+            - -c
+            - bin/blockscout eval "Elixir.Explorer.ReleaseTasks.create_and_migrate()" && bin/blockscout start
+          image: docker.io/blockscout/blockscout:5.1.0
+          volumeMounts:
+            - mountPath: /app/logs
+              name: blockscout-logs-vol
+          envFrom:
+            - configMapRef:
+                name: blockscout-env
+          ports:
+            - containerPort: 4000
+              name: blockscout
+        - name: postgres
+          command: ["docker-entrypoint.sh"]
+          args:
+            - -c
+            - max_connections=200
+          env:
+            - name: POSTGRES_HOST_AUTH_METHOD
+              value: trust
+            - name: POSTGRES_PASSWORD
```

Comment:
Does this value get populated later? I thought `POSTGRES_PASSWORD` couldn't be empty or null to run the postgres docker image.

From https://hub.docker.com/_/postgres
```
POSTGRES_PASSWORD
This environment variable is required for you to use the PostgreSQL image. It must not be empty or undefined.
```

Reply:
I copied this from [blockscout docker-compose setup](https://github.com/blockscout/blockscout/blob/master/docker-compose/services/docker-compose-db.yml). I believe based on reading docs, it's allowed to be an empty string (what this creates) when the auth method is set to trust. It will only allow localhost connections in this manner as well.